includes/classes/Command.php

```diff
@@ -283,10 +283,11 @@ private function put_mapping_helper( $args, $assoc_args ) {
 					 */
 					do_action( 'ep_cli_put_mapping', $indexable, $args, $assoc_args );
 
-					if ( $result ) {
+					if ( ! is_wp_error( $result ) ) {
 						WP_CLI::success( esc_html__( 'Mapping sent', 'elasticpress' ) );
 					} else {
-						WP_CLI::error( esc_html__( 'Mapping failed', 'elasticpress' ), false );
+						/* translators: Error message */
+						WP_CLI::error( sprintf( esc_html__( 'Mapping failed: %s', 'elasticpress' ), $result->get_error_message() ), false );
 
 						return false;
```

Comment:
To make it more readable, can we split these functions into different lines?
```suggestion
						/* translators: Error message */
						WP_CLI::error(
							sprintf(
								esc_html__( 'Mapping failed: %s', 'elasticpress' ),
								$result->get_error_message()
							)
						);
```